src/Request.php

```diff
@@ -201,7 +201,7 @@ protected function get_validation_rules(): array {
 		$validation_rules = GraphQL::getStandardValidationRules();
 
 		$validation_rules['require_authentication'] = new RequireAuthentication();
-		$validation_rules['disable_introspection']  = new DisableIntrospection();
+		$validation_rules['disable_introspection']  = new DisableIntrospection( 1 );
```

Comment:
Not sure about this, as we're overloading the `DisableIntrospection::isEnabled()` function which is the only thing that uses the constructor param.

Reply:
The `\GraphQL\Validator\Rules\DisableIntrospection` class in graphql-php now requires an argument. 

Without this we get phpstan error:

```
 204    Class WPGraphQL\Server\ValidationRules\DisableIntrospection constructor invoked with 0 parameters, 1 required.  
         🪪  arguments.count  
```

I suppose we could update the extending class as well, but this was an easy change to satisfy the need and not change behavior

Reply:
@justlevine I opened this PR to refactor how we're handling the DisableIntrospection instantiation: https://github.com/wp-graphql/wp-graphql/pull/3290